app/models/variable_lookup_key.rb

```diff
@@ -0,0 +1,18 @@
+class VariableLookupKey < LookupKey
```

Comment:
No, those paths are usually just added to the autoload paths.  Have a look at any of the existing ones, they don't prefix it as you describe.